package.json

```diff
@@ -36,6 +36,9 @@
         "cordova-android": ">=3.6.0",
         "cordova-windows": ">=4.4.0"
       },
+      "<5.0.4": {
+        "cordova-ios": "<=5.1.1"
```

Comment:
`"<6.0.0"` might be better here. Currently this would cause issues for any future patch release of 5.1.x.

Reply:
I applied suggestion